collectors/channels_collector.go

```diff
@@ -317,6 +340,150 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 			)
 		}
 	}
+
+	// Get all remote policies
+	remotePolicies, err := c.getRemotePolicies(getInfoResp.IdentityPubkey)
+	if err != nil {
+		channelLogger.Error(err)
+		return
+	}
+
+	// Export the inbound fee metric for a series of amounts.
+	var feeAmount btcutil.Amount = 100000
```

Comment:
Should this be named `amountToSend` instead? (If I understand correctly this is not a fee).

Reply:
It is the amount for which we want to calculate the fee. The `feeAmount` :)

Reply:
Had to read the code twice to get it fully :)

Reply:
Renamed to `receiveAmt`